measures/ResStockArguments/measure.rb

```diff
@@ -781,6 +773,22 @@ def run(model, runner, user_arguments)
     return true
   end
 
+  def get_max_heat_cool_load_served(args):
+    load_served = []
+    if args[:heating_system_type] == 'Furnace'
```

Comment:
Likewise this would become `HPXML::HVACTypeFurnace` and the ones below can also be updated.